lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -752,7 +752,8 @@ int amf_app_handle_pdu_session_accept(
   smf_msg->msg.pdu_session_estab_accept.message_type.msg_type =
       PDU_SESSION_ESTABLISHMENT_ACCEPT;
   smf_msg->msg.pdu_session_estab_accept.pdu_session_type.type_val = 1;
-  smf_msg->msg.pdu_session_estab_accept.ssc_mode.mode_val         = 1;
+  smf_msg->msg.pdu_session_estab_accept.ssc_mode.mode_val =
+      (pdu_session_resp->selected_ssc_mode + 1);
```

Comment:
where do we pick/configure the ssc mode? cc @panyogesh

Reply:
Its copied from UE, as in  set_amf_smf_context() "smf_ctx->smf_proc_data.ssc_mode  = message->ssc_mode". Its send to sesssiond and replayed back to AMF.

Reply:
Should not we pick one from AMF configs? It should be up to the core to pick SSC mode as it does not have to support all modes. E.g., similar to how we pick integration protection.

Reply:
I will stamp and land this code to unblock the team, but can you create and track the issue for future extension for mobility support @panyogesh

Reply:
@ulaskozat : Thanks. I have raised a new task #10547 for configuration and selection logic of SSC mode.